Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -358,7 +357,31 @@ void IOWindow::CreateMainLayout()
 
   m_main_layout->addLayout(hbox, 2);
   m_main_layout->addWidget(m_expression_text, 1);
-  m_main_layout->addWidget(m_parse_text);
+
+  m_result_value->setTabKeyNavigation(false);
+  m_result_value->setItemDelegate(new InputStateDelegate(this, 0, [&](int row) {
+    const auto lock = m_controller->GetStateLock();
+    return m_reference->GetState<ControlState>();
+  }));
+  m_result_value->setColumnCount(1);
+  m_result_value->setRowCount(1);
+  m_result_value->horizontalHeader()->hide();
+  m_result_value->horizontalHeader()->setSectionResizeMode(0, QHeaderView::Fixed);
+  m_result_value->verticalHeader()->hide();
+  m_result_value->verticalHeader()->setDefaultSectionSize(
+      m_result_value->verticalHeader()->minimumSectionSize());
+  m_result_value->setEditTriggers(QAbstractItemView::NoEditTriggers);
+  m_result_value->setSelectionBehavior(QAbstractItemView::SelectRows);
+  m_result_value->setSelectionMode(QAbstractItemView::SingleSelection);
+  m_result_value->setColumnWidth(0, 64);
+  m_result_value->setRowHeight(0, 22);
+  m_result_value->setFixedSize(65, 23);
```

Comment:
Creating another `QTableWidget` just for this is pretty ugly. Maybe you can create a text box (or other simple widget) and break out the paint function of `InputStateDelegate`?